packages/intl-datetimeformat/src/core.ts

```diff
@@ -594,6 +645,54 @@ export function bestFitFormatMatcherScore(
   return score;
 }
 
+function dateTimeStyleFormat(
+  dateStyle: DateTimeFormatOptions['dateStyle'],
+  timeStyle: DateTimeFormatOptions['timeStyle'],
+  dataLocaleData: DateTimeFormatLocaleInternalData
+): Formats {
+  let dateFormat: Formats | undefined, timeFormat: Formats | undefined;
+  if (timeStyle !== undefined) {
+    timeFormat = dataLocaleData.timeFormat[timeStyle];
+  }
+  if (dateStyle !== undefined) {
+    dateFormat = dataLocaleData.dateFormat[dateStyle];
+  }
+
+  if (
+    dateStyle !== undefined &&
+    dateFormat !== undefined &&
+    timeFormat !== undefined
+  ) {
+    const format = {...dateFormat, ...timeFormat};
+    delete format.pattern;
+    delete format.pattern12;
+
+    const connector = dataLocaleData.dateTimeFormat[dateStyle];
+    format.pattern = connector
+      .replace('{0}', timeFormat.pattern)
+      .replace('{1}', dateFormat.pattern);
+    if (timeFormat.pattern12 !== undefined) {
+      format.pattern12 = connector
+        .replace('{0}', timeFormat.pattern12)
+        .replace('{1}', dateFormat.pattern);
+    }
+    format.skeleton = connector
```

Comment:
hmm can u explain why this is needed?

Reply:
The function parseDateTimeSkeleton [includes the skeleton in it's response](https://github.com/formatjs/formatjs/blob/main/packages/intl-datetimeformat/src/skeleton.ts#L23) so i was trying to maintain consistency. However, now that i'm thinking more about this, this is not a true skeleton because string literals will not be properly quoted. This is misleading and also not part of the spec so i'll remove it.